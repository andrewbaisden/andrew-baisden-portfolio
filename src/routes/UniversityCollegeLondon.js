import React from 'react';
import { Helmet } from 'react-helmet';
import Header from '../components/Header';
import Footer from '../components/Footer';
import PortfolioPageUCL from '../components/PortfolioPageUCL';
import CheckPortfolioPage from '../components/CheckPortfolioPage';
// import PortfolioCarouselUCL from '../components/PortfolioCarouselUCL';

const UniversityCollegeLondon = () => (
  <div>
    <div>
      <Helmet>
        <meta
          name="description"
          itemProp="description"
          content="A recruitment agency approached me with a Front-End Web Developer role working for the University College London, which I accepted. The University had a backlog of work and they needed someone else to come on board and assist them with it."
        />
        <title>University College London - Portfolio - Full Stack Developer | Andrew Baisden | London</title>
        <link rel="canonical" href="https://andrewbaisden.com/university-college-london" />
      </Helmet>
    </div>
    <CheckPortfolioPage />
    <Header />
    <PortfolioPageUCL companyBG="#4FA2B7" />
    {/* <PortfolioCarouselUCL /> */}
    <Footer />
  </div>
);

export default UniversityCollegeLondon;
