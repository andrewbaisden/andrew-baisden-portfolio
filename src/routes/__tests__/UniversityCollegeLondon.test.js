import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import UniversityCollegeLondon from '../UniversityCollegeLondon';

Enzyme.configure({ adapter: new Adapter() });

describe('<UniversityCollegeLondon />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<UniversityCollegeLondon />);
    expect(wrapper).toMatchSnapshot();
  });
});
