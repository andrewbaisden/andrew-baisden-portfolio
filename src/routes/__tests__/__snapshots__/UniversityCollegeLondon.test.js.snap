// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UniversityCollegeLondon /> renders and matches the snapshot 1`] = `ShallowWrapper {}`;
